### ISSUE TITLE
[AdaptiveCards- Designer]: Focus Order is not in sequential order for the header controls on the 'Designer Page'.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7282]
https://microsoft.visualstudio.com/OS/_workitems/edit/35922292

**Comment by vagpt** [comment]
Hi @carlos-zamora and @anna-dingler,

Could you please provide us any update regarding this bug as issue is still repro.

**Comment by anna-dingler** [comment]
Hi @vagpt,

I believe this issue should be closed since this behavior is by design. As you can see in the screenshot below, the header controls are split into two groups (left and right). Focus order should go through all left elements before the right elements.

<img width="794" alt="image" src="https://github.com/microsoft/AdaptiveCards/assets/98650930/e3cdc0ac-877d-4397-b041-0e9235e766d6">

Please let me know if you have any questions - thanks!

**Comment by vagpt** [comment]
Hi @anna-dingler,

As per your provided feedback, closing this issue.
